Makefile

```
@@ -165,9 +166,12 @@ ifneq ($(strip $(TAG)),)
 	@docker run \
 		--rm \
 		-e CGO_ENABLED=1 \
+		-e GOPRIVATE=github.com/KiiChain/* \
+		-e GOMODCACHE=/go/pkg/mod \
 		-e TM_VERSION=$(TM_VERSION) \
 		-e COSMWASM_VERSION=$(COSMWASM_VERSION) \
 		-v `pwd`:/go/src/kiichaind \
+		-v $(GOMODCACHE):/go/pkg/mod:ro \
```

checkmake (0.3.2)

[warning] Target body for "-v $(GOMODCACHE)" exceeds allowed length of 5 lines (9).

(maxbodylength)